Makefile

```diff
@@ -38,10 +38,8 @@ endif
 DOCKER_NAMESPACE := open-cluster-management
 DOCKER_REGISTRY := quay.io
 
-BROWSER ?= chrome
```

Comment:
We're already defaulting to chrome within the script if `BROWSER` isn't set. Cypress also by default runs in chrome.